com.unity.multiplayer.mlapi/Runtime/Core/NetworkUpdateLoop.cs

```diff
@@ -237,6 +241,26 @@ public static PlayerLoopSystem CreateLoopSystem()
         [RuntimeInitializeOnLoadMethod]
         private static void Initialize()
         {
+#if UNITY_EDITOR
+            EditorApplication.playModeStateChanged += stateChange =>
+            {
+                switch (stateChange)
+                {
+                    case PlayModeStateChange.EnteredPlayMode:
+                        InjectSystems();
+                        break;
+                    case PlayModeStateChange.ExitingPlayMode:
+                        UninjectSystems();
+                        break;
+                }
+            };
+#else
+            InjectSystems();
+#endif
+        }
```

Comment:
hmm, I think we should have a good comment block explaining what's going on here :P
will put that in too!

Reply:
done ✅